pkg/storage/snapshot/restore.go

```diff
@@ -78,6 +78,7 @@ type vmRestoreTarget struct {
 	controller *VMRestoreController
 	vmRestore  *snapshotv1.VirtualMachineRestore
 	vm         *kubevirtv1.VirtualMachine
+	restoredVM *kubevirtv1.VirtualMachine
```

Comment:
Does this have to be a member of this struct? If it cannot be assigned when the `vmRestoreTarget` is created I think it would probably be better to pass it around on the stack

Reply:
done